tests/Fixer/FunctionNotation/NativeFunctionInvocationFixerTest.php

```diff
@@ -69,6 +69,53 @@ public function providerInvalidConfigurationElement()
         );
     }
 
+    public function testConfigureResetsExclude()
```

Comment:
Hehe, yes - let me take a look at this as soon as I have time.

What do you think about this PR so far, though, is there anything else you think should be done?